docs/users/Versioning.mdx

```diff
@@ -6,74 +6,6 @@ title: Versioning
-
-We publish a canary release on every successful merge to `main`, so **you never need to wait for a new stable version to make use of any updates**.
-
-Additionally, we promote to the `latest` tag on NPM once per week, **on Mondays at 1 pm Eastern**.
-
-The latest version under the `latest` tag is:
-
-<a href="https://www.npmjs.com/package/@typescript-eslint/parser">
-  <img
-    src="https://img.shields.io/npm/v/@typescript-eslint/parser/latest.svg?style=flat-square"
-    alt="NPM Version"
-  />
-</a>
-
-The latest version under the `canary` tag **(latest commit to `main`)** is:
-
-<a href="https://www.npmjs.com/package/@typescript-eslint/parser">
-  <img
-    src="https://img.shields.io/npm/v/@typescript-eslint/parser/canary.svg?style=flat-square"
-    alt="NPM Version"
-  />
-</a>
-
-:::note
-The only exception to the automated publishes described above is when we are in the final phases of creating the next major version of the libraries - e.g. going from `1.x.x` to `2.x.x`.
-During these periods, we manually publish `canary` releases until we are happy with the release and promote it to `latest`.
-:::
-
-### ESLint
-
-> The version range of ESLint currently supported is `^6.0.0 || ^7.0.0 || ^8.0.0`.
-
-We generally support at least the latest two major versions of ESLint.
-
-### Node
-
-This project makes an effort to support Active LTS and Maintenance LTS release statuses of Node according to [Node's release document](https://nodejs.org/en/about/releases).
-Support for specific Current status releases are considered periodically.
-
-### TypeScript
-
-> The version range of TypeScript currently supported is `>=4.3.5 <5.2.0`.
-
-Note that we mirror [DefinitelyTyped's version support window](https://github.com/DefinitelyTyped/DefinitelyTyped/#support-window) - meaning we only support versions of TypeScript less than 2 years old.
-
-You may find that our tooling works on older TypeScript versions however we provide no guarantees and **_we will not accept issues against unsupported versions_**.
-
-#### Supporting New TypeScript Releases
-
-With each new TypeScript release we file an issue to track the changes in the new version. The issue should always be pinned, and you can also [find the issues by searching for issues tagged with "New TypeScript Version"](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3A%22New+TypeScript+Version%22+sort%3Acreated-desc). If the issue is open, we do not have official support yet - please be patient.
-
-In terms of what versions we support:
-
-- We do not support the `beta` releases.
-- We _generally_ do not officially support the `rc` releases.
-- We endeavor to support the latest stable TypeScript versions as soon as possible after the release.
-
-Generally we will begin working on supporting the next release when the `rc` version is released.
```

Comment:
Ah, there's information in this page that's not in the other one. Could you move them over please?

Just skimming:

* Mentioning the DefinitelyTyped support window
* Not accepting issues against unsupported versions
* Not supporting beta or rc versions, but trying to support stables soon after release
* Having a _Version Warning Logs_ heading that can be linked to

Reply:
Oh, sorry, I missed this! I'll bring it back

---
One small question:

> Having a Version Warning Logs heading that can be linked to

This info is present [here](https://typescript-eslint.io/users/dependency-versions/#typescript)

https://github.com/typescript-eslint/typescript-eslint/blob/54ef459e0250a7d0d3ddad4ce1a458d003126383/docs/users/Dependency_Versions.mdx#L29-L31

Should I place it on `Versioning` page too?

Reply:
Or I need to add this heading to `Dependency_Versions` and add link to it from `Versioning`?

Reply:
Yeah, the info is there but a heading isn't. The heading is nice for discoverability and so folks can link directly to it.

Reply:
To be clear, I'm saying it's fine to delete this all out of `Versioning.mdx`, just put the `#### Version Warning Logs` above the same existing info in `Dependency_Versions.mdx`. That way we stick with having one place for the info.